Sources/MeiliSearch/Indexes.swift

```diff
@@ -392,7 +399,7 @@ public struct Indexes {
    */
   public func deleteDocument(
     _ documentId: String,
-    _ completion: @escaping (Result<Task, Swift.Error>) -> Void) {
+    _ completion: @escaping (Result<TaskInfo, Swift.Error>) -> Void) {
```

Comment:
Same as above, should this not be added in the `Tasks` PR ?

Reply:
Not really. These small chunks are very time-consuming to create a PR with only these changes. Sorry about that!